fmriprep/interfaces/images.py

```diff
@@ -122,6 +122,56 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class ConformSeriesInputSpec(BaseInterfaceInputSpec):
+    t1w_list = InputMultiPath(File(exists=True), mandatory=True,
+                              desc='input T1w images')
+
+
+class ConformSeriesOutputSpec(TraitedSpec):
+    t1w_list = OutputMultiPath(exists=True, desc='output T1w images')
+
+
+class ConformSeries(SimpleInterface):
+    input_spec = ConformSeriesInputSpec
+    output_spec = ConformSeriesOutputSpec
+
+    def _run_interface(self, runtime):
+        import nibabel as nb
+        import nilearn.image as nli
+        from nipype.utils.filemanip import fname_presuffix, copyfile
+
+        orig_imgs = [nb.load(fname) for fname in self.inputs.t1w_list]
+        reoriented = [nb.as_closest_canonical(img) for img in orig_imgs]
+        target_shape = np.max([img.shape for img in reoriented], axis=0)
+        target_zooms = np.min([img.header.get_zooms()[:3] + (1,) for img in reoriented], axis=0)
+
+        resampled_imgs = []
+        for img in reoriented:
+            zooms = np.array(img.header.get_zooms()[:3] + (1,))
+            shape = img.shape
+            if not (np.allclose(zooms, target_zooms) and shape == tuple(target_shape)):
+                scale_factor = target_zooms / zooms
+                shape_factor = target_shape.astype(float) / np.array(shape)
+                target_affine = np.diag(scale_factor).dot(img.affine)
+                target_affine[:3, 3] = img.affine[:3, 3] * shape_factor
```

Comment:
This shape factor is the remaining issue. The idea here is to scale the (x, y, z) origin to the new shape, so that it's proportionally in the same place. I think I'm doing it wrong, but haven't had time to think carefully about what should be done instead.